serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -17,19 +20,19 @@ Workflows in the development profile are not tailored for production environment
 
 The development profile is the easiest way to start playing around with Workflows and the operator.
 
-To get started, you can use an editor of your choice to create a new `KogitoServerlessWorkflow` Custom Resource YAML definition. For example:
+To get started, you can use an editor of your choice to create a new `SonataFlow` Custom Resource YAML definition. For example:
```

Comment:
How do we hook this up with the kn-workflow CLI?  Because my assumption is that users in general do NOT have to write CRs themselves.  It's fine for more expert people to understand this, or even write CRs themselves, but in general I assume kn-workflow will do that work for the rest.  This content most likely will live in another chapter, but I think we should point people there as the preferred option.
Action item for when the kn-workflow guide updates land?

Reply:
As soon as we get the kn cli docs, we can update it here and add a link there. Something like "Use kn workflow to deploy the CRs for you, this chapter is just an explanation of how to do it manually..."